code/API_definitions/location-subscriptions.yaml

```diff
@@ -0,0 +1,465 @@
+      summary: 'Operation to retrieve a list of event subscriptions - could be an empty list.'
+      operationId: getEventSubscriptionList
+      description: Operation to retrieve a list of event subscriptions
+      responses:
+        '200':
+          description: The list of event subscriptions is retrieved.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/EventSubscriptionInfo'
+        '400':
+          $ref: '#/components/responses/Generic400'
+        '401':
+          $ref: '#/components/responses/Generic401'
+        '403':
+          $ref: '#/components/responses/Generic403'
+        '500':
+          $ref: '#/components/responses/Generic500'
+        '503':
+          $ref: '#/components/responses/Generic503'
+  
+  /event-subscriptions/{eventSubscriptionsId}:
+    get:
+      tags:
+        - Location Event Subscription
+      summary: 'Operation to retrieve an event subscription'
+      operationId: getEventSubscription
+      description: Retrieve a given event subscription by ID
+      parameters:
+        - name: eventSubscriptionsId
+          in: path
+          description: event subscription identifier that was obtained from the create event subscription operation
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          description: Contains information about Event Subscriptions
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/EventSubscriptionInfo'
+        '400':
+          $ref: '#/components/responses/Generic400'
+        '401':
+          $ref: '#/components/responses/Generic401'
+        '403':
+          $ref: '#/components/responses/Generic403'
+        '404':
+          $ref: '#/components/responses/Generic404'
+        '500':
+          $ref: '#/components/responses/Generic500'
+        '503':
+          $ref: '#/components/responses/Generic503' 
+    delete:
+      tags:
+        - Location Event Subscription
+      summary: 'Operation to delete a event subscription'
+      operationId: deleteEventSubscription
+      description: Delete a given event subscription by ID
+      parameters:
+        - name: eventSubscriptionsId
+          in: path
+          description: delete a given event subscription by ID
+          required: true
+          schema:
+            type: string
+      responses:
+        '204':
+          description: Event subscription deleted
+        '400':
+          $ref: '#/components/responses/Generic400'
+        '401':
+          $ref: '#/components/responses/Generic401'
+        '403':
+          $ref: '#/components/responses/Generic403'
+        '404':
+          $ref: '#/components/responses/Generic404'
+        '500':
+          $ref: '#/components/responses/Generic500'
+        '503':
+          $ref: '#/components/responses/Generic503'
+  
+  /eventNotifications:
```

Comment:
uri's are in lowercase with hyphen

actually I would add as callback within POST /event-notifications... I've commented on a similar PR in CAMARA, if we change there we should change it here too.